bin/generate_tests.py

```diff
@@ -167,6 +167,15 @@ def to_roc_tuple(values: Any):
     list_content = ", ".join([to_roc(v) for v in tuple(values)])
     return f"({list_content})"
 
+def to_roc_record(obj: Dict[str, Any]):
+    items = []
+    for key, value in obj.items():
+        camel_key = to_camel(key)
+        roc_value = to_roc(value)
+        items.append(f"{camel_key}: {roc_value}")
+
+    return "{ " + ", ".join(items) + " }"
+
```

Comment:
Note: the order of dicts is guaranteed since Python 3.7, so this code is fine, but if you ever need to generate stable output with sets, you must iterate over `sorted(mySet)`.

Reply:
Good to know!